randomized-runner/src/test/java/com/carrotsearch/randomizedtesting/generators/TestRealisticUnicodeGenerator.java

```diff
@@ -2,6 +2,6 @@
 
 public class TestRealisticUnicodeGenerator extends StringGeneratorTestBase {
   public TestRealisticUnicodeGenerator() {
-    super(new UnicodeGenerator());
+    super(new RealisticUnicodeGenerator());
```

Comment:
maybe typo?

Reply:
Indeed, I'll fix!

Reply:
Thanks! (just FYI, my github account is "mocobeta"; not "macobeta" :cat2:)

Reply:
Darn. See - eyes see what they want to see...

Reply:
(well... the phrase is somehow catchy and stuck in my head for hours...)